index.src.html

```diff
@@ -761,7 +759,7 @@
 
 ### Attestation Models ### {#attestation-models}
       
-	<p>FIDO 2.0 specifies multiple attestation models:</p>
+	<p>WebAuth specifies multiple attestation models:</p>
```

Comment:
Pending resolution of the other discussion on abstracting away attestation details, can we say something like "WebAuth supports..." instead of "WebAuth specifies..."?